app/viewmodels/models/TaskListViewModel.scala

```diff
@@ -79,4 +79,8 @@ object TaskListStatus {
   case object Completed extends TaskListStatus("tasklist.completed")
 
   case object Updated extends TaskListStatus("tasklist.updated")
+
+  case object Reported extends TaskListStatus("tasklist.reported")
+
+  case class Reported(amount: String) extends TaskListStatus(Message("tasklist.reportedWithAmount", amount))
```

Comment:
Would it be better to keep `amount` as Int and just detect `0` when constructing the message and display `None` in that case?

Reply:
That was my initial approach, but I ran into hurdles with the implementation - I don't remember the exact circumstances, but it was something to do with having a combination of `case object`, `case class`, and/or `object` with an `apply` method that would carry out that `0` detection. There was always at least one error, and any fix I found to solve one error would result in a different error somewhere else.

Since it was beginning to get time-consuming, I opted for this implementation instead - I appreciate `amount: String` isn't great, but I think in this instance it's not too bad, because we don't do any further calculations to the value after it's generated, so we don't have to convert back and forth between `Int` and `String` multiple times, and deal with any of the headaches that might entail.

Reply:
Not sure what those hurdles are, but I was thinking that if we do something like `case class Reported(amount: String) extends TaskListStatus(Message("tasklist.reportedWithAmount", if (amount == 0) "None" else amount.toString))
` that would simplify the calling code.
I.e.
`if (numReported == 0) Reported("None") else Reported(numReported.toString)` can be replaced with `Reported(numReported)`

Reply:
Ah okay, I see what you mean - thanks for the suggestion, I've just implemented this and pushed the change!